src/plugins/intel_cpu/src/nodes/kernels/scaled_attn/softmax_kernel.hpp

```diff
@@ -203,7 +189,7 @@ inline void scale_add2_reduce_max(float* a,
         }
 
         if (has_attn_mask) {
-            auto v_mask = _mm512_loadu_ps(attn_mask + i);
+            auto v_mask = mm512_uni_loadu_ps(attn_mask + i);
```

Comment:
I am wondering will it safe some time if we will convert attn mask to f32 once before attn_softmax?

Reply:
Since the `attn_mask` is shared among different transformer blocks, it's timesaving to only convert `attn_mask` to f32 once. PR https://github.com/openvinotoolkit/openvino/pull/18865 may help to convert `attn_mask` globally.